.ci/jobs/elastic+apm-server+6.7+beats-updated.yml

```diff
@@ -0,0 +1,40 @@
+---
+- job:
+    name: elastic+apm-server+6.7+beats-updated
```

Comment:
can the name be parameterized too?  Ideally this check would run against 7.1 also

Reply:
@graphaelli we've to replace this job with a multibranch pipeline to avoid to have to modify the job with each new branch.

Reply:
also probably is time to talk about how to automate the beats update process.